tests/filecheck/dialects/llvm/generic-syntax.mlir

```diff
@@ -0,0 +1,61 @@
+// RUN: xdsl-opt %s --print-op-generic | filecheck %s
```

Comment:
```suggestion
// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | filecheck %s
```
To check that it roundtrips as well

Reply:
This is the command `XDSL_GENERIC_ROUNDTRIP` right?